src/Endatix.Api/Setup/EndatixAppExtensions.cs

```diff
@@ -0,0 +1,36 @@
+using Endatix.Framework.Hosting;
+using FastEndpoints;
+using FastEndpoints.Swagger;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Endatix.Setup;
+
+/// <summary>
+/// Provides extension methods for configuring and setting up the Endatix application
+/// </summary>
+public static class EndatixAppExtensions
+{
+    /// <summary>
+    /// Adds the API Endpoints associated provided by the Endatix app
+    /// </summary>
+    /// <param name="endatixApp"></param>
+    /// <returns>An instance of <see cref="IEndatixApp"/> representing the configured application.</returns>
+    public static IEndatixApp AddApiEndpoints(this IEndatixApp endatixApp)
+    {
+        endatixApp.Services
+                .AddCors()
+                .AddFastEndpoints()
+                .SwaggerDocument(o =>
+                    {
+                        o.ShortSchemaNames = true;
+                        o.DocumentSettings = s =>
+                        {
+                            s.Version = "v0";
+                            s.DocumentName = "Internal MVP (Alpha) Release";

```

Comment:
A small thing we've commented in a previous PR but somehow it reached here - the internal MVP ;)

Reply:
@dspetrov  - this was outdated. I have merged latest from `main` and you can see this file has not been changed. Sorry for that.